keyboards/alps64/keymaps/xyverz/keymap.c

```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _DVORAK 0
+#define _COLEMAK 1
+#define _QWERTY 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  DVORAK = SAFE_RANGE,
+  COLEMAK,
+  QWERTY,
+  ADJUST
+};
+
+// Useful defines
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_DVORAK] = LAYOUT_aek_103( \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
Just on the top line, or all lines of the keymap definition?

Reply:
All of them. They're only needed for the layout `#define`, as they tell the preprocessor to ignore the newline character which would ordinarily end the define, so we can format them nicely.